espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/viewaction/RootViewFinder.java

```diff
@@ -0,0 +1,45 @@
+package io.appium.espressoserver.lib.viewaction;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.matcher.ViewMatchers.isRoot;
+
+/**
+ * Get the Root View of the Android App
+ */
+public class RootViewFinder {
```

Comment:
this class implementation is quite hacky and cryptic. Would it be possible to add more documentation here?